### ISSUE TITLE
Copy of Role with implied hidden content has keyboard focus - Section display

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5337]
Sites identified: MCO
Page(s) identified: Media object page, section display

WCAG compliance level: A

Description:

Some elements have a role that implies their internal structure should be hidden from assistive technologies. In other words, their sub-elements have a presentational role.

If a sub-element has keyboard focus, it should be moved outside of the presentation role.

Screen readers will ignore any content with a presentation role. If this includes focusable sub-elements, then these will not be announced to the user who might miss out on important information.

Learn more in [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=38100866150&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r90/failed/WkdFdk1xZS8=;NGE3MDcyNzU=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaDlnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f6f1c0bf4fc4de6919ce643f8fac769f386cbfc9/image.png)
